config/config.php

```diff
@@ -24,6 +24,11 @@
         $config = ArrayUtils::merge($config, include $file);
     }
 
+    // Development mode enabled
+    if (file_exists(__DIR__ . '/../config/development.config.php')) {
+        $config = ArrayUtils::merge($config, require __DIR__ . '/../config/development.config.php');
+    }
```

Comment:
I would place this *after* the block that caches configuration; we likely should *never* cache development configuration.